### PR TITLE
[settings] restore show EXIF picture information setting

### PR DESCRIPTION
**Pull request by popcornmix**
## Description

This reverts a change from https://github.com/xbmc/xbmc/pull/7992

## Motivation and Context

See: http://forum.kodi.tv/showthread.php?tid=304156
Loading exif information is quite expensive in both time and memory used.
Users with large photo libraries or running on low powered hardware may prefer to avoid this.

## How Has This Been Tested?
It has been in milhouse/newclock5 builds for ages

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by jjd-uk** [comment]
For what it's worth this has my +1 as forced reading the exif metadata makes the opening of a folder containing a large pics very slow. For a photo enthusiast, having a folder containing the photo's from a vacation can easily be 500+ photos in size.

**Comment by fritsch** [comment]
jenkins build this please

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -20410,3 +20410,13 @@ msgstr ""
...
 msgid "Virtual filesystems"
 msgstr ""
+
+#. Description of setting "Pictures -> Show EXIF picture information" with label #38207
+#: system/settings/settings.xml
+msgctxt "#38207"
+msgid "Show EXIF picture information"
+msgstr ""
+
+msgctxt "#38208"
```
Comment (MartijnKaijser) [inline addons/resource.language.en_gb/resources/strings.po]: @popcornmix please also add description and location here. Before merging and backporting (can be separate commit)

**Comment by popcornmix** [comment]
Added description for help text string

**Comment by MartijnKaijser** [comment]
Jenkins build this please

**Comment by jjd-uk** [comment]
@popcornmix reminder that if you've got the time a backport for Krypton would be appreciated.
